lib/core/services/user_service.dart

```diff
@@ -116,7 +120,7 @@ class UserService {
           switch (attendanceStatus) {
             case AttendanceStatus.NotYetCheckedIn:
               responseData = await Util.request(RequestType.PUT, urlRecordAttendance, {
-                'in': DateTime.now().toIso8601String(),
+                'in': currentDateTimeString,
```

Comment:
pls convert to Dart's DateTime first, then use toIso8601String() to push to firebase